tests/utils.py

```diff
@@ -21,6 +21,18 @@
     'source': 'test'
 }
 
+POSTGRES_RAW_DOC = {
```

Comment:
Is there any way this could be merged with the real raw_doc? If not that's fine, just curious